### DISCUSSION TITLE
ICMP Host Fingerprinting Through TTL Analysis | Caleb Finigan

### DISCUSSION THREAD

**Discussion by giscus** [CalebFIN/CalebFIN.github.io#1] [General]
# ICMP Host Fingerprinting Through TTL Analysis | Caleb Finigan

ICMP isn’t just for ping and traceroute; it’s also a valuable tool in security research.

http://127.0.0.1:4000/posts/icmp-host-fingerprinting-through-ttl-analysis/

**Comment by CalebFIN** [comment]
Thanks for Reading, 
```this is really just a test post...```
